src/app/domains/Question/components/QuestionForm/QuestionForm.js

```diff
@@ -50,116 +91,80 @@ function QuestionForm(props) {
 
   return (
     <>
-      <Row noGutters mb={2}>
-        <Col>
-          <Card style={styles.cardStyle}>
+      <Row noGutters mb={2} height="inherit">
+        <Col v="center" order={2}>
+          <Card
+            style={{
```

Comment:
It's better to make all necessary computations outside of the template (render function)